### PR TITLE
Fixing deprecated notices from Twig 1.27

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

**Comment by Koc** [comment]
You need increase minimal supporting twig version inside composer for this changes

**Comment by kubawerlos** [comment]
@Koc thanks for the notice.

**Comment by stof** [comment]
We still have a few failures there:

- some unexpected calls to mocks, which should be debugged
- some remaining Twig deprecations, which means your commit is incomplete

**Comment by hvt** [comment]
That second deprecation notice (`Passing a string as the $source argument of Twig_Environment::tokenize() is deprecated since version 1.27. Pass a Twig_Source instance instead.`) must be fixed in Assetic itself. There is already a PR for that one: https://github.com/kriswallsmith/assetic/pull/819

**Comment by kubawerlos** [comment]
@stof all what was possible fixed, but [this one](https://github.com/kriswallsmith/assetic/pull/820) must go first.

File: .travis.yml [hunk 1]
```diff
@@ -38,4 +38,4 @@ install:
     - if [ "$TRAVIS_PHP_VERSION" = "hhvm" ]; then composer remove --dev --no-update kriswallsmith/spork; fi;
     - composer update $COMPOSER_FLAGS
 
-script: phpunit -v
+script: vendor/bin/phpunit -v
```
Comment (kubawerlos) [inline .travis.yml]: To my surprise all checks have passed after I moved it back (with only PHPUnit warnings).

**Comment by kubawerlos** [comment]
I have used dev-master for assetic to see the tests, after it is tagged I think this will be ready to merge.

**Comment by hvt** [comment]
@stof would be great if you could tag new versions of both assetic and assetic-bundle, when this one is merged.

**Comment by kubawerlos** [comment]
@hvt you mean tag assetic, then merge this one with a version >=1.3.3 and tag this, right?

@stof I have prepared release notes you can use: https://github.com/kubawerlos/assetic/releases/tag/v1.3.3

**Comment by hvt** [comment]
@kubawerlos that's what I mean yes :+1:

**Comment by kubawerlos** [comment]
@stof can you tag kriswallsmith/assetic?

File: composer.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
         "symfony/dependency-injection": "~2.3|~3.0",
         "symfony/console": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
-        "kriswallsmith/assetic": "~1.3"
+        "kriswallsmith/assetic": "dev-master#e01016358b205cff01c57cd4cebeec31c74383da"
```
Comment (stof) [inline composer.json]: please revert this

Comment (kubawerlos) [inline composer.json]: done

**Comment by kubawerlos** [comment]
@stof can we go with this?

**Comment by kubawerlos** [comment]
@stof can you merge this and create a tag?

**Comment by craue** [comment]
I also just stumbled upon the `Twig_Node::getLine` and `Twig_TokenStream::getFilename` deprecations. Will this PR be merged soon?

**Comment by stof** [comment]
Thank you @kubawerlos.

**Comment by hvt** [comment]
Great, thanks @stof and @kubawerlos. A new tag would be even greater ;].

**Comment by stof** [comment]
@hvt I'm on it

**Comment by fraggger** [comment]
@stof Hi, are there any news for this issue?

**Comment by xabbuh** [comment]
@fraggger This PR was part of the 2.8.1 release.

**Comment by fraggger** [comment]
@xabbuh I use 2.8.1, and even dev-master has this issue
`The Twig_Node::getLine method is deprecated since version 1.27 and will be removed in 2.0. Use getTemplateLine() instead`

**Comment by xabbuh** [comment]
Are you sure that this is coming from AsseticBundle?

**Comment by fraggger** [comment]
I run codeception tests and they appear. And AsseticBundle was mentioned in this issue [https://github.com/symfony/symfony/issues/20367](url) 
What I did so far as quickfix: I removed error message in Twig_Node class for getLine() method for now,

**Comment by stof** [comment]
@fraggger the fact that it was mentionned at some point does not mean that this message is triggered by AsseticBundle in your own project. Any other project proving a Twig tag could trigger the same one.

**Comment by fraggger** [comment]
@stof You are of course right. It's difficult to find out, which 3rd party bundle uses this deprecated method. Guess I have to wait for a fix :/

**Comment by stof** [comment]
@fragger locate the URL triggering this deprecation, open it in your browser, then access the web profiler to see the deprecation warning. The profiler displays a stack trace for deprecation warnings

**Comment by fraggger** [comment]
@stof Longer story: I use jenkins for deployment. When the code is deployed, i run codeception with PhantomJS. All tests are successfull and before PhantomJS can be stopped, this error appears:

    [sshexec] Time: 4.45 seconds, Memory: 58.00MB
    [sshexec] 
    [sshexec] OK (5 tests, 13 assertions)
    [sshexec] - HTML report generated in file:///var/www/myProject/state_2017-01-20_15-45-44/tests/_output/report.html
    [sshexec] - XML report generated in file:///var/www/myProject/state_2017-01-20_15-45-44/tests/_output/report.xml
    [sshexec] 
    [sshexec] Remaining deprecation notices (2)
    [sshexec] 
    [sshexec] The Twig_Node::getLine method is deprecated since version 1.27 and will be removed in 2.0. Use getTemplateLine() instead: 2x
    [sshexec]     2x in 
    [sshexec] Application::run from Codeception
    [sshexec] 
    [sshexec] Stopping PhantomJS Server
    [sshexec] .
    [sshexec] 
    [sshexec] PhantomJS server stopped
    
    BUILD FAILED
    /var/lib/jenkins/workspace/myProject/build.xml:71: The following error occurred while executing this line:
    /var/lib/jenkins/workspace/myProject/build.xml:78: The following error occurred while executing this line:
    /var/lib/jenkins/workspace/myProject/build.xml:157: Remote command failed with exit status 1

Line 157 in my build.xml:
`vendor/codeception/codeception/codecept run --xml --html --steps --debug`

**Pull request merged**